### ISSUE TITLE
Minimum viable product

### ISSUE THREAD

**Issue by AriHrannar** [veritus/veritus-backend#44]
Features that I can see in the MVP
- [x] Signup
- [x] Login
- [ ] Logout
- [x] Create a promise (Admin)
- [ ] Add tags to promise (both during and after creation) (Admin)
- [x] See all cases
- [ ] Add tags to cases (Admin)
- [ ] See all politicians
- [ ] See promises made by politician
- [x] See all political parties
- [ ] See promises made by political party
- [ ] Dashboard (Politician leaderboard, Political party leaderboard, more?)
- [ ] Search

Other features 
- [ ] Login using Íslykill
- [ ] See connected cases on promise (share 80% of the tags or something)
- [ ] See suggested connected cases on promise (share 50% of the tags or something)
- [ ] See connected promises on case (share 80% of the tags or something)
- [ ] See suggested connected promises on case (share 50% of the tags or something)
- [ ] See all districts
- [ ] See all politicians in district
- [ ] See all parties in district
- [ ] Politician leaderboard in each district

**Comment by skabbi** [comment]
You are web scraping some data from althingi.is, correct? Shouldn't a list of what needs to be scraped be part of the MVP?

On a side note, should the same process be acting as an API server and web scraper, or should they be split into separate projects?

**Comment by AriHrannar** [comment]
Yes it should be there, ill go over it again when im done with docker.

I would like to keep everything in the same project / place for now. If needed it can be split up later.
